TestName,Result,Message
test_Tainted_SO_cwe094_0.TestCWE094_0.test_correctness,failure,[] is not an instance of <class 'dict'>
test_Tainted_SO_cwe094_0.TestCWE094_0.test_vulnerability,success,
